salt/personalised-covers/aws.sls

```diff
@@ -0,0 +1,10 @@
+aws-credentials:
+    file.managed:
+        - name: /home/{{ pillar.elife.deploy_user.username }}/.aws/credentials
+        - source: salt://personalised-covers/config/home-user-.aws-credentials
+        - user: {{ pillar.elife.deploy_user.username }}
```

Comment:
Okay added that in below, I did add it to its own file (referencing the same pillar variables though)